src/python/WMCore/WMSpec/StdSpecs/DataProcessing.py

```diff
@@ -17,6 +17,17 @@ class DataProcessing(StdBase):
     for this kind of WMSpec.
     """
 
+    def __init__(self):
```

Comment:
I think in this case, we have ignore pylint. It seems all other Specs has the same structure and if we change only this one. It looks more puzzled when we look at the code.